### PR TITLE
fix(curriculum): replace `->` with actual right arrow symbol

### PR DESCRIPTION
**Pull request by huyenltnguyen**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

We're using a hyphen and a greater than symbol to represent a right arrow (`->`). Visually this is okay, but screen readers would announce this as "greater than".

This PR changes the combination to an actual right arrow (screen readers would announce this as "right arrow").

Alternatively, I think we could just say "From X to Y" and get rid of the symbol.

### CODE REVIEW

**Comment by bbsmooth** [comment]
My testing on Windows shows that NVDA announces this as "right arrow" and JAWS and Narrator announce this as "rightward arrow". So screen readers do seem to announce this symbol properly. I don't have a mac or iPhone so I can't test VoiceOver but I'm assuming it is probably OK.

I think the question is, as @huyenltnguyen pointed out, should we rely on the arrow symbol here, or use actual text to communicate to screen reader users what is being displayed? Let's consider Task 1 in Learn Conversation Starters in the Break Room. If I was reading these examples out loud to someone, I would probably not say "right arrow". I would probably say something like "The statement ... becomes the question ...". Or if you want to use the same wording as in the sentence that introduces these examples, something like: "The statement ... turns into the question ...". 

I'm not suggesting we add all of this hidden text, but I think I would be leaning towards at least adding alt text for the arrow, probably going with "turns into the question" for these specific examples. I also think [WCAG SC 1.1.1](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html#dfn-non-text-content) technically requires alt text for the arrow since unicode characters are generally considered to be non-text content.

Another option might be to turn this into a table of two columns, one for the statement and one for the question. Then you wouldn't need the arrow at all and I think it would be clear to everyone what is being presented, with no additional helper text needed.

| Statement         | Question     |
|--------------|-----------|
|He is here.      |Is he here?   |
|They are ready.      |Are they ready?  |

**Comment by moT01** [comment]
what about just commas: `` `git clone`, `git add`, `git commit`, `git push` ``
result: `git clone`, `git add`, `git commit`, `git push`

**Comment by bbsmooth** [comment]
I like Tom's idea. I was playing around with a few possibilities over the weekend and I pretty much came to the same conclusion. The comma will cause a screen reader to give a slight pause, so I think a screen reader user would be able to distinguish each command.

**Comment by moT01** [comment]
I gave this a quick revisit. I think that, yea, for something like the git quiz, the commas work - e.g: `git clone`, `git add`, `git commit`, `git push`

But for the English lessons on this PR, I would be fine with the table idea, but just adding more text works as well - e.g:
`He is here.` becomes `Is he here?`
`They are ready.` becomes `Are they ready?`

Either way works. Dealers choice.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b05367b59300bcb5f18ef.md [hunk 1]
```diff
@@ -14,8 +14,9 @@ When you want to ask such questions, you just swap the positions of the subject
 
 Here are some more examples of turning statements into questions using `to be`: 
 
-`He is here.` -> `Is he here?`
-`They are ready.` -> `Are they ready?`
+`He is here.` → `Is he here?`
+
+`They are ready.` → `Are they ready?`
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b05367b59300bcb5f18ef.md:19]:
```suggestion
Affirmative: `He is here.` 
Interrogative: `Is he here?`

Affirmative: `They are ready.` 
Interrogative: `Are they ready?`
```
Thank you for catching that. For the new curriculum we are not using arrows anymore. What do you think of this suggestion so we can make it more descriptive?

File: curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b69e10d6606a0185d4d4f.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ dashedName: task-23
 
 So far you have learned how to create questions with the verb `to be`. In sentences with the verb `to be` (am, is, are) you just have to change the order of the noun and the verb to create a question. Like this:
 
-`You are a developer` -> `Are you a developer?`
+`You are a developer` → `Are you a developer?`
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b69e10d6606a0185d4d4f.md:14]:
```suggestion
Affirmative: `You are a developer` 
Interrogative: `Are you a developer?`
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657cd962b45f48127c183d7b.md [hunk 3]
```diff
@@ -13,8 +13,9 @@ As you already studied, For most verbs conjugation can be simple. You just  add
 
 For example:
 
-`You talk` -> `He talks`
-`They do` -> `It does`
+`You talk` → `He talks`
+
+`They do` → `It does`
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657cd962b45f48127c183d7b.md:18]:
```suggestion
Verb `to talk` conjugation:

`I talk`, `you talk`, `we talk`, and `they talk`. 
`He talks`, `she talks`, and `it talks`.

Verb `to do` conjugation:

`I do`, `you do`, `we do`, and `they do`. 
`He does`, `she does`, and `it does`.
```
This one I had to be more creative. But the principle was the same. Removing the arrows and making them more descriptive.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657dcee413423174ca3747f5.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ dashedName: task-112
 
 To elaborate a question using `there is` you just need to change the order of the verb `to be`, like so:
 
-`There is an ATM nearby` -> `Is there an ATM nearby?`
+`There is an ATM nearby` → `Is there an ATM nearby?`
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657dcee413423174ca3747f5.md:12]:
```suggestion
Affirmative: `There is an ATM nearby` 
Interrogative: `Is there an ATM nearby?`
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/657b1dfec76149836ea5c7d0.md [hunk 5]
```diff
@@ -13,9 +13,9 @@ There are some verbs you can't use `didn't` to create negative sentences in the
 
 If you want to create a negative sentence using the verb `to be` in the past, simply  add `not` after `was` or `were`. For example:
 
-`It was working normally` -> `It wasn't working normally`.
+`It was working normally` → `It wasn't working normally`.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/657b1dfec76149836ea5c7d0.md:16]:
```suggestion
Affirmative: `It was working normally` 
Negative: `It wasn't working normally`.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/657b1dfec76149836ea5c7d0.md [hunk 6]
```diff
@@ -13,9 +13,9 @@ There are some verbs you can't use `didn't` to create negative sentences in the
...
 
-`They were informed earlier` -> `They weren't informed earlier`.
+`They were informed earlier` → `They weren't informed earlier`.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/657b1dfec76149836ea5c7d0.md:18]:
```suggestion
Affirmative: `They were informed earlier`
Negative: `They weren't informed earlier`.
```

**Review by nieldakarla — CHANGES_REQUESTED** [review]
Thank you for bringing  up this issue with the arrows. I've made a review in which I've entirely removed the arrows and changed the examples to be more descriptive, and more in line with the standards we are using for the new curriculum.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b05367b59300bcb5f18ef.md (hunk 1 above)
Comment (nieldakarla) [reply curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b05367b59300bcb5f18ef.md:19]: @huyenltnguyen I think this option is excellent. I wasn't aware we could add tables. If you make this changes they would definitely work.

**Review by nieldakarla — APPROVED** [review]

**Review by Sembauke — APPROVED** [review]

**Pull request merged**